src/guide/typescript-support.md

```diff
@@ -133,6 +133,39 @@ const Component = defineComponent({
 })
 ```
 
+### Augmenting Types for Use with Plugins or User Definitions
+
+Vue 3 provides `globalProperties` property for [plugins](./plugins.html#writing-a-plugin) to add to Vue’s global properties in each component instance. In some cases, we also need to [do it by ourselves](../api/application-config.html#globalproperties):
+
+```typescript
+// User Definition
+const app = Vue.createApp()
+app.config.globalProperties.foo = 'bar'
```

Comment:
**suggestion**: we try to avoid abstract meaningless names for variables in docs. Would you mind changing `foo` and `bar` to something more 'real-life'? 😅